www/source/partials/docs/_internals-bootstrapping.html.md.erb

```diff
@@ -5,164 +5,339 @@ This document provides developer documentation on how the Habitat system becomes
 
-## Part I: Setup
+## Part I: Background
 
 In order to bootstrap the system from scratch, you should be familiar with how the Linux From Scratch project works.
 
 We add the following software to augment the Linux From Scratch toolchain:
 
-* Statically built [BusyBox](https://www.busybox.net/) - used for the unzip implementation
-* Statically built [Wget](https://www.gnu.org/software/wget/) with OpenSSL support - used by the build program to download sources
-* A copy of curl’s [cacert.pem](https://curl.haxx.se/ca/cacert.pem) certificates - used by wget when connecting to SSL-enabled websites
+* Statically built [BusyBox](https://www.busybox.net/) - used for the unzip implementation. This is copied in to tarball as `/tools/bin/busybox`.
+* Statically built [Wget](https://www.gnu.org/software/wget/) with OpenSSL support - used by the build program to download sources. This is copied in to the tarball as `/tools/bin/wget.real`.
+* Statically built [rq](https://github.com/dflemstr/rq) - used by the build program to verify configuration value types. This is copied in to the tarball as `/tools/bin/rq`.
+* A symlink of `unzip` which points to the BusyBox binary and used by the build program to extract zipped source archives. This is copied in to the tarball as `/tools/bin/unzip -> busybox`.
+* A copy of curl’s [cacert.pem](https://curl.haxx.se/ca/cacert.pem) certificates - used by wget when connecting to SSL-enabled websites. This is copied into the tarball as `/tools/etc/cacert.pem`.
+* A small wrapper script for `wget` so that we use the SSL certificates we added to the tarball. Its source is:
+
+```shell
+#!/tools/bin/busybox sh
+exec /tools/bin/wget.real --ca-certificate /tools/etc/cacert.pem $*
+```
 
 Finally, we place a recent last-known-good copy of the `hab` binary inside `tools/bin`.
 
-The entire tarball of bootstrap "tools" lives inside the [stage1 studio](https://habitat-studio-stage1.s3.amazonaws.com/habitat-studio-stage1-20160612022150.tar.xz) tarball. This should be unpacked into `/tools` on a Linux host that will serve as the build environment until the system is self-sustaining through the rest of this procedure.
+The entire tarball of bootstrap "tools" lives inside the [stage1 studio](http://s3-us-west-2.amazonaws.com/habitat-studio-stage1/habitat-studio-stage1-20180312233639.tar.xz) tarball. This should be unpacked into `/tools` on a Linux host that will serve as the build environment until the system is self-sustaining through the rest of this procedure.
+
+Note: In order to build large amounts of softwar in the fastest way possible, we recommend you use a powerful machine, such as an `c4.4xlarge` on Amazon Web Services (AWS). While it may be possible to run these builds in Docker containers, in workstation virtual machines, or older hosts, it's well worth it to spend a few dollars to save potentially dozens of hours of wait time.
 
 ## Part II: Stage 0
 
+### Creating The stage1 tarball
```

Comment:
The header says "Stage 0", but the subsequent text talks about "Stage 1"; which is correct?